pkg/interceptors/interceptors.go

```diff
@@ -168,3 +171,31 @@ func (i *interceptor) nodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 
 	return handler(ctx, req)
 }
+
+func (i *interceptor) createVolume(ctx context.Context, req *csi.CreateVolumeRequest,
+	info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (res interface{}, resErr error) {
+
+	lock, err := i.opts.locker.GetLockWithID(ctx, req.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	if closer, ok := lock.(io.Closer); ok {
+		defer closer.Close()
+	}
+
+	if !lock.TryLock(i.opts.timeout) {
+		return nil, status.Error(codes.Aborted, pending)
+	}
+	defer lock.Unlock()
+
+	volumeLabels := make(map[string]string)
+
+	// volumeLabels = GetVolumeLabels()
```

Comment:
Please remove this dead code?

Reply:
added a comment for now, will replace it with sidecar call.